src/components/services/EdgeCoreManager.js

```diff
@@ -34,7 +34,8 @@ const contextOptions: EdgeContextOptions = {
     }
   },
 
-  plugins: allPlugins
+  plugins: allPlugins,
+  authServer: 'https://login-test.edge.app/api'
```

Comment:
Remove this and merge, or make it configurable from `env.json` (alternatively, `src/theme/appConfig.js`